target/linux/brcm47xx/image/Makefile

```diff
@@ -403,7 +419,7 @@ endef
 TARGET_DEVICES += linksys-wrt54g
 
 define Device/linksys-wrt54gs
-  DEVICE_TITLE := Linksys WRT54GS
+  DEVICE_MODEL := WRT54GS
```

Comment:
Please add the v1 here, as there is a v4 for the subsequent device. (At least that's how I understood the response on the general question?)